### DISCUSSION TITLE
Needs to get data from Google Authenticator app and not have to create a new set accounts.

### DISCUSSION THREAD

**Discussion by bondo01** [Authenticator-Extension/Authenticator#1132] [Ideas]
This is basically worthless if the user already has over 50 2FA accounts. User is not going to create all new 3FA accounts to use this extension.  This needs to import the exiting accounts or links to the REAL Google Authenticator app.

**Comment by Sneezry** [comment]
> You can also import Google Authenticator accounts. In Google Authenticator, tap on the three dots in the upper right-hand corner of the screen. Tap on Transfer Accounts. Tap on Export Accounts. Verify your identity. Choose the account information you wish to transfer from the list. Tap on Next. Take a screenshot to save the QR image (iPhone), or take a picture with another phone/camera (Android). Then use Import QR Image Backup to import the accounts.
>
> Google Authenticator supports exporting up to 10 accounts at one time. However, when importing to Authenticator Extension it is recommended to export five accounts at one time to prevent errors.

https://authenticator.cc/docs/en/export-and-import

**Reply by spinozaNNN** [reply]
I suggest including these instructions in the extension.  I had to come here to find them.
Thanks for your good work!
